### PR TITLE
[Shropshire] Add map for tilma assets

### PR DESCRIPTION
**Pull request by MorayMySoc**
Commit with initial asset mapping

### CODE REVIEW

File: layers/shropshire.map [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+      "wfs_srs"           "EPSG:3857 EPSG:27700" ## REQUIRED
+      "gml_include_items" "site_code,central_asset_id" ## Optional (serves all attributes for layer)
+      "gml_central_asset_id_alias" "central_as"
+      "gml_featureid"     "central_asset_id" ## REQUIRED
+      "wfs_enable_request" "*"
+    END
+    TYPE POINT
+    STATUS ON
+    CONNECTIONTYPE OGR
+    CONNECTION 'shropshire/CTGR'
```
Comment (davea) [inline layers/shropshire.map]: Should this be `CTGR.gpkg`?

File: layers/shropshire.map [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+    FORMATOPTION "FORM=SIMPLE"
+  END
+
+  LAYER
+    NAME "Street_Gazeteer"
+    METADATA
+      "wfs_title"         "Street_Gazeteer" ##REQUIRED
+      "wfs_srs"           "EPSG:3857 EPSG:27700" ## REQUIRED
+      "gml_include_items" "ASSET_ID, SITE_CODE" ## Optional (serves all attributes for layer)
+      "gml_central_asset_id_alias" "central_as"
```
Comment (davea) [inline layers/shropshire.map]: As mentioned on Slack this can be removed (here and in other layers).

**Review by davea — CHANGES_REQUESTED** [review]
Couple of little things is all

File: layers/shropshire.map [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+  OUTPUTFORMAT
+    NAME "geojson"
+    DRIVER "OGR/GEOJSON"
+    MIMETYPE "application/json; subtype=geojson"
+    FORMATOPTION "STORAGE=stream"
+    FORMATOPTION "FORM=SIMPLE"
+  END
+
+  LAYER
+    NAME "Street_Gazeteer"
```
Comment (davea) [inline layers/shropshire.map]: Should this be `Street_Gazetteer` (typo) instead? (Sorry, should have spotted this sooner!)

**Review by davea — APPROVED** [review]
One query but looks good 👍

**Review by dracos — APPROVED** [review]
Assuming they all work, looks good :)

**Comment by MorayMySoc** [comment]
Merged into staging

**Pull request closed without merging**